.github/workflows/tests.yml

```diff
@@ -0,0 +1,25 @@
+name: Tests
+
+on: [push, pull_request]
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-versions: ['7.4', '8.0', '8.1', '8.2']
```

Comment:
It can stay like this, but does it make any sense to test on 8.0 and 8.1? Probably we're never going to run this code on those versions?

Reply:
I did think about removing them, but given this is available on Packagist I thought best to keep them tested as someone else may be running them

Reply:
Ah that makes sense, sorry I forgot about that :sweat_smile: